<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GmiForm</class>
 <widget class="QWidget" name="GmiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Прогноз гидрометеорологических условий по району</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtParamsBox" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="1">
       <widget class="QDateEdit" name="dtDateBegin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbRequestId">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Номер:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="leRequestId"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Срок прогноза:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="3">
       <widget class="QComboBox" name="cbLength">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Краткосрочный (36 ч.)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Среднесрочный (3 сут.)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Долгосрочный (7 сут.)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="cbTimeBegin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>00:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18:00</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="StationWidget" name="stationSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="isMapButtonVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isSearchVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isCoordVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isSavedStationsVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isHorisontal" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectUsersBtn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Выбор пользователей</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sendUsersCheck">
        <property name="text">
         <string>Отправка пользователям</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="confirmButtons">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
